site/app/controllers/PollController.php

```diff
@@ -472,6 +472,16 @@ public function submitEdits(): RedirectResponse {
                 return new RedirectResponse($returnUrl);
             }
         }
+        if ($_POST["removed_responses"] !== "") {
+            $prev_responses = $poll->getResponses();
+            $removed_resp = explode(",", $_POST["removed_responses"]);
+            foreach ($removed_resp as $response_id) {
+                if (array_key_exists($response_id, $prev_responses)) {
+                    $this->core->addErrorMessage("Error occured in editing poll: attempt to delete response option that has already been submitted as an answer");
+                    return new RedirectResponse($returnUrl);
+                }
+            }
+        }
```

Comment:
I can still send a back end request and delete an option that was not intended. When I made that request I also got quite a few errors too.